src/python/doc/nerve_gic_complex_ref.rst

```diff
@@ -12,3 +12,10 @@ Cover complexes reference manual
    :show-inheritance:
 
    .. automethod:: gudhi.CoverComplex.__init__
+
+.. autoclass:: gudhi.NGIComplex
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
+   .. automethod:: gudhi.NGIComplex.__init__
```

Comment:
You should remove these lines as it is documented in `src/python/doc/cover_complex_sklearn_isk_ref.rst`

Reply:
Thanks! Done.